subsys/app_event_manager/app_event_manager_shell.c

```diff
@@ -7,6 +7,7 @@
 #include <stdlib.h>
 #include <zephyr/shell/shell.h>
 #include <app_event_manager.h>
+#include <caf/events/button_event.h>
```

Comment:
I'm realizing this creates a backward dependency from `app_event_manager` to `CAF`

Maybe it would be better to create a CAF shell. Sorry I misled you on this one. If you go this route then of course you can ignore my comment about integrating into the `SHELL_STATIC_SUBCMD_SET_CREATE` call, and just move your additions to their own file somewhere in the CAF library

Though in that case you would ideally still use `SHELL_STATIC_SUBCMD_SET_CREATE` and make your button sim command a subcommand of the CAF shell root command

Reply:
creating a CAF shell is what I was anticipating...

Reply:
> creating a CAF shell is what I was anticipating...

I'm only suggesting a skeletal one with a single command. I don't think that is any less reasonable than adding a command supporting a single CAF event to the app_event_manager shell

Reply:
> creating a CAF shell is what I was anticipating...

Yeah, sorry about that. I encouraged Tony to add a command to this existing shell because it already exists, but forgot that `app_event_manager` and `CAF` are technically distinct, and the former is a dependency of the latter.